client/src/components/lobby/PublicLobbyList.tsx

```diff
@@ -148,18 +148,18 @@ function useLobbies(appId: string): LobbyV3[] {
   const [lobbies, setLobbies] = React.useState<LobbyV3[]>([]);
   React.useEffect(() => {
     if (appId) {
-      hathoraSdk.lobbyV3.listActivePublicLobbies().then(({ classes }) => {
-        if (classes != null) {
-          setLobbies(classes);
+      hathoraSdk.lobbiesV3.listActivePublicLobbies().then((l) => {
+        if (l != null) {
```

Comment:
Minor: Can we rename this param to something more descriptive like `lobbies`?

Reply:
sure